fedimintd/src/fedimintd.rs

```diff
@@ -300,9 +304,7 @@ async fn run(
     // TODO: Fedimintd should use the config gen API
     // on each run we want to pass the currently passed password, so we need to
     // overwrite
-    if let Some(password) = opts.password {
```

Comment:
Do we have any protection against overwriting old password? Is it going to get overwritten every time now?

Reply:
No protection to overwrite old password. Will need to check if a password exists before setting a new one.

> Do we have any protection against overwriting old password? Is it going to get overwritten every time now with a new random one?